.github/workflows/ci.yml

```diff
@@ -0,0 +1,35 @@
+name: CI Check
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    name: Build Docker Image
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Get short SHA
+        id: sha
+        run: |
+          echo ::set-output name=sha7::$(git rev-parse --short=7 ${{ github.sha }})
+      - name: Get current time

```

Comment:
im a bit uncomfortable with the province here, is there no other action, or even just a run step that we can use here?

Reply:
I'll spend more time searching. If there isn't a built-in from GitHub for this, I'll update it to use something built into ubuntu.

Reply:
no reason to bring in more platform specific steps. accomplished the same thing with ubuntu's date